hedera-node/test-clients/src/main/resource/contract/contracts/Module049OpcodesExist/Module049OpcodesExist.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.8.24;
+
+contract Module049OpcodesExist {
+    function try_transient_storage() public {
+        assembly {
+            tstore(10, 11111)
+            pop(tload(10))
```

Comment:
Q: is there a way to confirm the expected details popped match what was stored?

Reply:
(see above comment at TLOAD/TSTORE.)